modules/influxdb/build.gradle

```diff
@@ -0,0 +1,15 @@
+description = "Testcontainers :: InfluxDB"
+
+ext {
+    influxdbJavaClientVersion = '2.9'
+
+    junitVersion = '4.12'
+}
+
+dependencies {
+    compile project(':testcontainers')
+
+    compile "org.influxdb:influxdb-java:${influxdbJavaClientVersion}"
```

Comment:
1) `compileOnly`, users should bring their own drivers
2) could you please specify the version here? `ext` doesn't make a lot of sense unless shared between the modules (not the case)